ui/src/pages/GlobalHome.js

```diff
@@ -0,0 +1,293 @@
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.learnHow)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderDevelopersCard() {
+    return (
+      <Card className="globalhome-thirdpartycard">
+        <div>
+          <h3 className="mdc-typography--headline3">
+            <FormattedMessage {...messages.developers} />
+          </h3>
+          <p className="mdc-typography--body1">
+            <FormattedHTMLMessage {...messages.developersHowToHelp} />
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.getStarted)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderThirdPartyHelpCards() {
+    return (
+      <div className="globalhome-thirdpartywrapper">
+        <div className="endbars-headerline-wrapper" dir="ltr">
+          <span className="mdc-typography--overline endbars-headerline">
+            <FormattedMessage {...messages.youCanHelp} />
+          </span>
+        </div>
+        <Grid>
+          <Row>
+            <Cell desktopColumns={6}>{this.renderRespondersCard()}</Cell>
+            <Cell desktopColumns={6}>{this.renderDevelopersCard()}</Cell>
+          </Row>
+        </Grid>
+      </div>
+    );
+  }
+
+  renderRepoList() {
+    var cells = this.state.repos.map(repo => (
+      <Cell key={repo.repoId}>
+        <RepoCard repoInfo={repo} />
+      </Cell>
+    ));
+    return <Grid><Row>{cells}</Row></Grid>;
+  }
+
+  render() {
+    if (this.state.error) {
+      return <div>An error occurred</div>
+    }
+    if (!this.state.isLoaded) {
+      return <LoadingIndicator />;
+    }
+    return (
+      <div className="globalhome-wrapper">
+        {this.renderHeader()}
+        {this.renderRepoList()}
+        {this.renderThirdPartyHelpCards()}
+        <Footer />
+      </div>
+    );
+  }
+}
+
+/**
+ * RepoCard is a component for a card in the repo list.
+ */
+
+const repoCardMessages = defineMessages({
+  repoRecordCount: {
+    id: 'RepoCard.repoRecordCount',
+    defaultMessage: 'Tracking {recordCount} records.',
+    description: ('A message displaying how many data records we have in the '
+        + 'database.'),
+  },
+});
+
+class RepoCardImpl extends Component {
+  constructor(props) {
+    super(props);
+    this.goTo = this.goTo.bind(this);
+  }
+
+  goTo() {
+    this.props.history.push("/" + this.props.repoInfo.repoId);
```

Comment:
Can be done later, but don't we need special handling for a case where the app root path is not "/", as in our production (https://google.org/personfinder/)?

Reply:
Yeah, I am going to need to do something about that. I haven't dug too far into what's involved, but it doesn't look like it'll be too bad (React/react-router do have support for that kind of thing).